robotbase/src/main/java/com/team2357/frc2022/subsystems/IntakeSubsystem.java

```diff
@@ -16,13 +18,15 @@
 public class IntakeSubsystem extends ClosedLoopSubsystem {
     public DoubleSolenoid m_intakeSolenoid;
     private VictorSPX m_intakeVictor;
+    private ArduinoSensor m_inputIRSensor;
 
     /**
      * @param intakeVictor Victor SPX to use to control intake
      */
     public IntakeSubsystem(VictorSPX intakeVictor, DoubleSolenoid intakeSolenoid) {
         m_intakeSolenoid = intakeSolenoid;
         m_intakeSolenoid.set(Value.kOff);
+        m_inputIRSensor = new ArduinoSensor(Constants.ARDUINO.ARDUINO_SENSOR_DEVICE_NAME, Constants.ARDUINO.IR_SENSOR_JSON_NAME);
```

Comment:
This call will be irrelevant, but we should not be using constants in a subsystem. Subsystems are rarely purely implementation specific, so we would take these as either config or constructor parameters to make the code more reusable.